sdk/core/azure-core-tracing-opencensus/README.md

```diff
@@ -19,7 +19,7 @@ documentation][api_documentation] | [Samples][samples]
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-core-tracing-opencensus</artifactId>
-  <version>1.0.0-beta.4</version>
+  <version>1.0.0-beta.5</version>
```

Comment:
@JimSuplizio should this be annotated for the updating script?

Reply:
annotated with the update script